<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FoldersCompare</class>
 <widget class="QDialog" name="FoldersCompare">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>742</width>
    <height>556</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FoldersCompare</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Select two folders to compare:</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Folder 1:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListView" name="listView"/>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>Select folder</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Folder 2:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListView" name="listView_2"/>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_2">
         <property name="text">
          <string>Select folder</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Duplicate files matches:</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QListView" name="listView_3"/>
       </item>
       <item>
        <widget class="QListView" name="listView_4"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_3">
       <property name="text">
        <string>Compare</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
